website/blog/2023-12-08-silver.md

```diff
@@ -0,0 +1,585 @@
+---
+author: Tomasz Godzik
+title: Metals v1.1.1 - Silver
+authorURL: https://twitter.com/<login>
+authorImageURL: https://github.com/<login>.png
+---
+
+We're happy to announce the release of Metals v1.1.1, which continues to improve
+reporting mechanisms and is finally feature complete when it comes to Scala 3.
```

Comment:
```suggestion
reporting mechanisms and is finally feature complete when it comes to Scala 3.
```

I think it might be worth explaining what is meant by "feature complete when it comes to Scala 3". Surely I don't assume this means that there will be no new features related to Scala 3 right? Metals will always continue to improve and add new things. I'm assuming this instead means that you've reached feature parity with Scala 2 support?

Reply:
Changed, should make sense now, let me know what you think!